doc/uftrace-recv.md

```diff
@@ -27,6 +27,37 @@ Data will be sent using `uftrace-record` with -H/\--host option.
 :   Run given (shell) command as soon as receive data.  For example, one can run "uftrace replay" for received data.
 
 
+EXAMPLE
+=======
+The uftrace recv command should be run before sends data by record command.
+
+    # HOST 
+    $ uftrace recv -d recv_data --port 1234
+
+Above command start server with port by given(if you not specify port by `--port` option, it will work at default `8090`.) to receive data from remote client. 
+
+    # CLIENT :
+    $ uftrace record -H localhost -d example_data --port 1234 example
+
+Above command sends the trace data to a remote server that pointed by given -H option (`localhost` in this case) after running the example program. and as you see, you can choose save directory by using `-d` option(sending data will save to `example_data` in this case).
```

Comment:
I prefer two spaces between sentences.  Also it needs to be started with an upper case letter.  And please put a space before the parenthesis.